.github/workflows/e2e-tests-chrome.yml

```diff
@@ -0,0 +1,76 @@
+name: Cypress Tests (Chrome)
+on:
+  - push
+jobs:
+  chrome:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.5
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - name: Setup Ruby 2.6
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.5
+      - name: Setup Node 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Install PostgreSQL 11 client
+        run: sudo apt-get -yqq install libpq-dev
+      - name: Build App
+        env:
+          PGHOST: localhost
+          PGUSER: postgres
+          RAILS_ENV: development
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          KNOWN_TEACHER_UUID: ff37bd17-42be-47ab-8141-2f5e3b7f2b7f
+          API_TOKEN: fb5542df-50e7-4c05-a74e-4f2123f46554
+        run: |
+          gem install bundler
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+          npm config set //npm.pkg.github.com/:_authToken=$GITHUB_TOKEN
+          npm config set @dfe-digital:registry=https://npm.pkg.github.com
+          bin/rails db:setup
```

Comment:
If you do change the env this runs under to `test` then db:seed won't run as it checks it's not running in test to avoid polluting the test db. Makes me think a ci env is needed.